packages/labs/gen-wrapper-vue/goldens/test-element-a/src/ElementWithoutProps.vue

```diff
@@ -0,0 +1,33 @@
+<script setup lang="ts">
+import {h, useSlots, reactive} from 'vue';
+import {assignSlotNodes, Slots} from '@lit-labs/vue-utils/wrapper-utils.js';
+import '@lit-internal/test-element-a/element-without-props.js';
+
+export interface Props {}
+
+const vueProps = defineProps<Props>();
+
+const defaults = reactive({} as Props);
+const vDefaults = {};
+
+let hasRendered = false;
+
+const slots = useSlots();
+
+const render = () => {
+  const eventProps = {};
+
+  const props = eventProps as typeof eventProps & Props;
+  for (const p in vueProps) {
+    const v = vueProps[p as keyof Props];
+    if (v !== undefined || hasRendered) {
+      (props[p as keyof Props] as unknown) = v ?? defaults[p as keyof Props];
+    }
+  }
```

Comment:
It actually seems like an empty `Props` interface means the whole `vueProps` and code looping through it might be unnecessary too?

Reply:
i did think this too but wasn't sure if some vue magic needed this stuff to exist

i'll update 👍